examples/Example.h

```diff
@@ -436,6 +436,11 @@ struct WgWindow : Window
         //Set the canvas target and draw on it.
         verify(canvas->target(instance, surface, width, height));
     }
+
+    void refresh() override 
+    {
+        wgpuSurfacePresent(surface);
```

Comment:
@SergeyLebedkin It looks like this is closer to an engine change. `sync()` seems more appropriate to me since the example sw/gl actually refreshes the SDL window, not their context.